functional/README.md

```diff
@@ -10,6 +10,23 @@ Since the tests utilize [`systemd-nspawn`][systemd-nspawn], this needs to be inv
 
 If the tests are aborted partway through, it's currently possible for them to leave residual state as a result of the `systemd-nspawn` operations. This can be cleaned up using the `clean.sh` script.
 
+### Using go flags in functional tests
+
+`fleet/functional` scripts forwards all arguments directly to `go` binary. This allows you to pass `-run regexp` argument to the `./test` script and run specific test function. Example below shows how to run only `TestSchedule*` test functions:
```

Comment:
nits:
to **the** go binary
and run specific test function(s)
**The** example below